index.js

```diff
@@ -0,0 +1,52 @@
+const addBtn = document.querySelector('button');
+const title = document.querySelector('[data-title]');
+const author = document.querySelector('[data-author]');
+const displaySection = document.querySelector('[data-display]');
+
+let bookCollection = [];
+
+const books = localStorage.getItem('books');
+const allBooks = JSON.parse(books);
+if (books) {
+  bookCollection = allBooks;
+}
+
+const addBook = (title, author) => {
+  function Book(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  const newBook = new Book(title, author);
+
+  bookCollection.push(newBook);
```

Comment:
- [ ] [OPTIONAL]
It will be a good idea if we use a non-mutating array method such as `concat` over `push`. In this way, we avoid modifying the original array.